_includes/footer.html

```diff
@@ -1,59 +1,33 @@
+                    </a>
+                </div>
+                <div>
+                    <a href="https://cs.illinois.edu/" target="_blank">
+                        <img src="/assets/images/affiliations/uiuc.png" alt="UIUC Logo" style="vertical-align:middle; display:inline-block;">
+                    </a>
+                </div>
+                <div>
+                    <a href="https://www.cs.oberlin.edu/" target="_blank">
+                        <img src="/assets/images/affiliations/oberlin.png" alt="Oberlin College Logo" style="vertical-align:middle; display:inline-block;">
+                    </a>
+                </div>
+                <div>
+                    <a href="https://www.cs.washington.edu/" target="_blank">
+                        <img src="/assets/images/affiliations/uw.png" alt="UW CSE Logo" style="vertical-align:middle; display:inline-block;">
+                    </a>
+                </div>
             </div>
         </div>
-
-        <div class="footer-col-wrapper">
-            <table style="table-layout:fixed; width:100%; border: none;">
-                <tbody>
-                    <tr>
-                        <td style="width: 25%; border: none;">
-                            <a href="https://cse.ucsd.edu/" target="_blank">
-                                <img src="/assets/images/affiliations/ucsd.png" alt="UCSD CSE Logo" style="vertical-align:middle; display:inline-block;">
-                            </a>
-                        </td>
-                        <td style="width: 25%; border: none;">
-                            <a href="https://cs.illinois.edu/" target="_blank">
-                                <img src="/assets/images/affiliations/uiuc.png" alt="UIUC Logo" style="vertical-align:middle; display:inline-block;">
-                            </a>
-                        </td>
-                        <td style="width: 25%; border: none;">
-                            <a href="https://www.cs.oberlin.edu/" target="_blank">
-                                <img src="/assets/images/affiliations/oberlin.png" alt="Oberlin College Logo" style="vertical-align:middle; display:inline-block;">
-                            </a>
-                        </td>
-                        <td style="width: 25%; border: none;">
-                            <a href="https://www.cs.washington.edu/" target="_blank">
-                                <img src="/assets/images/affiliations/uw.png" alt="UW CSE Logo" style="vertical-align:middle; display:inline-block;">
-                            </a>
-                        </td>
-                    </tr>
-                </tbody>
-            </table>
-        </div>
-
     </div>
+</footer>
 
-</footer>
\ No newline at end of file
+<script src="/assets/bower_components/jquery/dist/jquery.min.js"></script>
+<script src="/assets/bower_components/vide/dist/jquery.vide.min.js"></script>
+<!-- <script src="/assets/js/auto-video.js"></script> -->
```

Comment:
Do we need all of the jquery files in the repo and also served by the webserver? I'd be inclined to just have the minified version unless we plan on editing jquery itself (which I never do).